src/components/HelpDialog.vue

```diff
@@ -99,42 +115,167 @@ export default defineComponent({
+    });
+
+    // VoicevoxのOSSライセンス取得
+
+    let licenses = ref<Record<string, string>[]>();
+    store.dispatch("GET_OSS_LICENSES").then((obj) => (licenses.value = obj));
+
+    let policy = ref<string>();
+    store.dispatch("GET_POLICY_TEXT").then((obj) => (policy.value = obj));
+
+    const pagedata = computed(() =>
+      (
+        [
+          {
+            type: "item",
+            name: "ソフトウェアの利用規約",
+            component: Policy,
+            props: {
+              policy: policy.value,
+            },
+          },
+          {
+            type: "item",
+            name: "音声ライブラリの利用規約",
+            component: LibraryPolicy,
+          },
+          {
+            type: "item",
+            name: "使い方",
+            component: HowToUse,
+          },
+          {
+            type: "item",
+            name: "開発コミュニティ",
+            component: OssCommunityInfo,
+          },
+          {
+            type: "item",
+            name: "ライセンス情報",
+            component: OssLicense,
+            props: {
+              licenses: licenses.value,
+            },
+          },
+          {
+            type: "item",
+            name: "アップデート情報",
+            component: UpdateInfo,
+            props: {
+              updateInfos: updateInfos.value,
+              isUpdateAvailable: isUpdateAvailable.value,
+              latestVersion: latestVersion.value,
+            },
+          },
+          {
+            type: "item",
+            name: "よくあるご質問",
+            component: QAndA,
+          },
+          {
+            type: "item",
+            name: "お問い合わせ",
+            component: ContactInfo,
+          },
+        ] as PageData[]
+      ).concat(
+        // エンジンが一つだけの場合は従来の表示のみ
+        (store.state.engineIds.length > 1
+          ? Object.values(store.state.engineManifests)
+          : []
+        ).flatMap((manifest) => [
+          {
+            type: "separator",
+            name: manifest.name,
+          },
+          {
+            type: "item",
+            name: "利用規約",
+            parent: manifest.name,
+            component: Policy,
+            props: {
+              policy: manifest.termsOfService,
+            },
+          },
+          {
+            type: "item",
+            name: "ライセンス情報",
+            parent: manifest.name,
+            component: OssLicense,
+            props: {
+              licenses: manifest.dependencyLicenses,
+            },
+          },
+          {
+            type: "item",
+            name: "アップデート情報",
+            parent: manifest.name,
+            component: UpdateInfo,
+            props: {
+              updateInfos: manifest.updateInfos,
+              // TODO: エンジン側で最新バージョンのチェックを実装したい。
+              //       /update_availableみたいなエンドポイントがあれば良さそう
+              isUpdateAvailable: false,
+            },
+          } /* TODO: エンジン毎に項目を追加する */,
+        ])
+      )
+    );
+
+    const selectedPage = ref(1);
```

Comment:
selectedPageIndexにすると更にわかりやすいかもと思いました。
```suggestion
    const selectedPageIndex = ref(1);
```

Reply:
変更しました。